Arm/State.lean

```diff
@@ -645,6 +690,13 @@ theorem read_mem_bytes_of_w :
     rw [n_ih]
   done
 
+theorem read_mem_bytes_w_of_read_mem_eq
+    (h : ∀ n addr, read_mem_bytes n addr s₁ = read_mem_bytes n addr s₂)
+    (fld val n₁ addr₁) :
```

Comment:
I'm curious why you chose to have explicit args here.

Reply:
I wanted to put the `h` hyp in front, to avoid accidentally capturing anything, and having implicit args after an explicit one felt a bit off.
Also, there are no other arguments from which `fld`, `val`, `n\1` or `addr\1` could be inferred, so having them explicit seems the right choice

Reply:
Thanks! That helps!